client/src/components/CategoryShow.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect } from "react";
+import PastaTile from "./PastaTile.js";
+
+const CategoryShow = (props) => {
+  const [category, setCategory] = useState({
+    name: "",
+    pastas: [null],
```

Comment:
We should be able to just define this state for pastas as an empty array, it shouldn't have a default `null` value within the array